yameducation/educationx-app-ios
# Yam Customizations

## Description

This pull request customizes the app for Yam education, by changing out assets, URLs, and configuration for Yam education's environment and branding.

## Supporting information

https://tasks.opencraft.com/browse/BB-7674

## Testing instructions

Clone the app, and follow the instructions in the README to load it into XCode and build it. Build for dev, and attempt to use the app.

The app should bring you to the Yam Auth0 login page when you open it. If you log in, you should then be logged into the app. Once logged in, many parts of the app will not work as the LMS still needs to be upgraded and an extension installed.

## Dependencies
